T/TMscore/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TMscore"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    FileSource("https://seq2fun.dcmb.med.umich.edu//TM-score/TMscore.cpp", "30274251f4123601af102cf6d4f1a9cc496878c1ae776702f554e2fc25658d7f"),
```

Comment:
I presume that when there is a new version they replace this file in-place, making this recipe non-reproducible, right?

Reply:
Sometimes people put the code in a repository and update the source in the repository when needed, at least we get a more reproducible source (as long as the repository isn't deleted)